posthog/models/plugin.py

```diff
@@ -171,7 +169,7 @@ class PluginType(models.TextChoices):
 
     name: models.CharField = models.CharField(max_length=200, null=True, blank=True)
     description: models.TextField = models.TextField(null=True, blank=True)
-    url: models.CharField = models.CharField(max_length=800, null=True, blank=True)
+    url: models.CharField = models.CharField(max_length=800, null=True, blank=True, unique=True)
```

Comment:
This is likely fine since we don't have that many plugins, just verified that all urls in US and EU are unique `select url, count(*) as cnt from posthog_plugin group by url order by cnt desc` which they weren't but I deleted the unused duplicate, so now they are.